modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,44 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array autoloaded data - autoload size, entries, option and its names.
+ */
+function perflab_aao_autoloaded_options() {
+	global $wpdb;
+
+	/**
+	 * Filters max bytes threshold to get options from Database.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param int $option_threshold Autoloaded options threshold size. Default 100.
+	 */
+	$option_threshold = apply_filters( 'perflab_aao_autoloaded_option_table_threshold', 100 );
```

Comment:
I suggest we align the name with the module and the other function names.
```suggestion
	$option_threshold = apply_filters( 'perflab_aao_autoloaded_options_table_threshold', 100 );
```